### ISSUE TITLE
Accompagner visuellement la progression dans les répliques

### ISSUE THREAD

**Issue by ipante** [maladesimaginaires/intnetviz#50]
Pour moins désorienter le lecteur lors de l'évolution du graphe, les répliques devraient présenter une forme de continuité dans leur mise en évidence, par exemple en faisant glisser le texte et sa mise en évidence.

**Comment by grandjeanmartin** [comment]
Oui, c'est effectivement une conséquence visuelle de cette mise en évidence, le saut doit être doux.
